src/oscar/core/context_processors.py

```diff
@@ -25,7 +25,7 @@ def metadata(request):
             'homepage_url': settings.OSCAR_HOMEPAGE,
             'language_neutral_url_path': strip_language_code(request),
             # Fallback to old settings name for backwards compatibility
-            'use_less': (getattr(settings, 'OSCAR_USE_LESS', None)
-                         or getattr(settings, 'USE_LESS', False)),
+            'use_scss': (getattr(settings, 'OSCAR_USE_SCSS', None)
+                         or getattr(settings, 'USE_SCSS', False)),
```

Comment:
I think we need to remove this variable and setting entirely - you can't load the scss in the browser any more so it isn't used.